packages/react-components/src/components/MessageThread.tsx

```diff
@@ -572,6 +574,20 @@ export type MessageThreadProps = {
    * Optional strings to override in component
    */
   strings?: Partial<MessageThreadStrings>;
+
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * @beta
+   * Optional function called when someone clicks on the file download icon.
+   */
+  fileDownloadHandler?: FileDownloadHandler;
```

Comment:
Why are we adding fileDownloadHandler as a separate prop when it is being handled by onRenderFileDownload Prop?